src/server/scripts/EasternKingdoms/zone_western_plaguelands.cpp

```diff
@@ -74,37 +95,71 @@ class npc_the_scourge_cauldron : public CreatureScript
 
             switch (me->GetAreaId())
             {
-                case 199:                                   //felstone
-                    if (player->GetQuestStatus(5216) == QUEST_STATUS_INCOMPLETE ||
-                            player->GetQuestStatus(5229) == QUEST_STATUS_INCOMPLETE)
+                case AREA_FELSTONE_FIELD:                                   //felstone
+                    if (player->GetQuestStatus(5216) != QUEST_STATUS_INCOMPLETE &&
+                        player->GetQuestStatus(5229) != QUEST_STATUS_INCOMPLETE)
+                        break;
+
+                        //A creature is summoned if not already present
+                    if (ObjectAccessor::GetCreature(*me, bilemawGUID))
+                        break;
+
+                    if (Creature* bilemaw = me->SummonCreature(NPC_CAULDRON_LORD_BILEMAW, 1728.6443f, -1174.7982f, 59.05936f, 2.356194496154785156f, TEMPSUMMON_TIMED_OR_DEAD_DESPAWN, 600000))
                     {
-                        me->SummonCreature(11075, 0.0f, 0.0f, 0.0f, 0.0f, TEMPSUMMON_TIMED_OR_DEAD_DESPAWN, 600000);
+                        bilemawGUID = bilemaw->GetGUID();
```

Comment:
I think this is pointless as the creature you interact with is destroyed and thus I believe the GUIDs are lost anyway.

Reply:
Summoned creatures do not despawn until they have passed the time and died. Despite the fact that the interacting creatures are destroyed. The guid is still there

Reply:
I seem to see what you mean, I'll go back later and see if the guid will be lost when the plague cauldron is destroyed